enterprise/api/v1/serializers.py

```diff
@@ -954,17 +954,38 @@ def validate(self, data):  # pylint: disable=arguments-differ
         return data
 
 
+class LicensesInfoSerializer(serializers.Serializer):
+    """
+    Nested serializer class to allow for many license info dictionaries.
+    """
+    email = serializers.CharField(required=False)
+    course_run_key = serializers.CharField(required=False)
+    license_uuid = serializers.CharField(required=False)
+
+    def create(self, validated_data):
+        return validated_data
+
+    def validate(self, data):  # pylint: disable=arguments-differ
+        missing_fields = []
+        LOGGER.info("MEMES MEMES MEMES MEMES")
```

Comment:
need to remove these logs?

Reply:
you can see me getting frustrated at debugging haha